packages/components/src/components/permissions/PermissionAssignments.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2018-2019 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
+import * as React from 'react';
+import { Button, Col, Panel, Row } from "react-bootstrap";
+import { List } from 'immutable'
+import { Security } from '@labkey/api'
+import { PermissionsProviderProps, Principal, SecurityPolicy, SecurityRole } from "./models";
+import { PermissionsRole } from "./PermissionsRole";
+import { PrincipalDetailsPanel } from "./PrincipalDetailsPanel";
+import { LoadingSpinner } from "../../components/base/LoadingSpinner";
+import { Alert } from "../../components/base/Alert";
+
+interface Props extends PermissionsProviderProps {
+    title?: string
+    containerPath: string
+    policy: SecurityPolicy
+    onChange: (policy: SecurityPolicy) => any
+    onSuccess: () => any
+    rolesToShow?: List<string>
+    typesToShow?: List<string>
```

Comment:
`typesToShow` and `rolesToShow` probably deserve a comment so it's more clear what the possible values are.

Reply:
done